plugins/actions/meson.build

```diff
@@ -0,0 +1,27 @@
+libaction_vala_sources = [
+  'action-listboxrow.vala',
+  'action-page.vala',
+  'actions.vala',
+  'action.vala',
+  'enums.vala',
+  'preferences-page.vala',
+]
+
+both_libraries(
+  'action',
+  libaction_vala_sources,
+  dependencies: gnome_pomodoro_lib_dep,
+  install: true,
+  install_dir: plugin_libdir,
+  vala_args: ['--gresources', join_paths(meson.current_source_dir(), 'resources', 'resources.xml')],
```

Comment:
This will not be rebuild when the resources change, try using [`files()`](http://mesonbuild.com/Reference-manual.html#files).